app/src/main/java/nyc/ignitelabs/civicduty/IntentUtility.kt

```diff
@@ -0,0 +1,13 @@
+package nyc.ignitelabs.civicduty
+
+import android.content.Context
+import android.content.Intent
+import nyc.ignitelabs.civicduty.view.EditActivity
+
+fun  addressEditIntent(context: Context, address: String ) : Intent {
```

Comment:
What's your thinking behind `IntentUtility` + anonymous functions? I can see three alternatives, off the top of my head:

1. Intents are built up during construction of the class meant to fire them, and assigned to properties.
2. Intents are built up lazily by class methods in the class meant to fire them.
3. Intents are generated by anonymous methods like these, but with narrower responsibilities---say, `nyc.ignitelabs.civicduty.GreetingIntentUtility`.

I feel like any one of the four (or more) possibilities can be justified, so it'd just be fun to get at your reasoning.

Reply:
Idea was to move this code outside of the Activity class for now. As we apply arch patterns, I think it'll end up being close to your 3rd point! Good eye man

Reply:
Totally makes sense. Thanks!